inst/schema/spec.md

```diff
@@ -92,7 +92,7 @@ Schema: [`Status.schema.json`](Status.schema.json)
         "[ data      ]  dat: 20 x 2",
         "[ start     ]  2017-09-12 09:11:03",
         "",
-        "> png(\"mygraph.png\")",
+        "> png("mygraph.png")",
```

Comment:
does this need quoting here?

Reply:
Not sure when/why this changed, reverted that.